.circleci/config.yml

```diff
@@ -39,6 +23,12 @@ jobs:
     working_directory: ~/node-10
     steps: *buildSteps
 
+  "node-12":
+    docker:
+      - image: circleci/node:12
+    working_directory: ~/node-12
+    steps: *buildSteps
+
 workflows:
   version: 2
   build:
```

Comment:
You need to remove `"node-6"` from the list below and add `"node-12"`  to trigger the builds, then I can assign them as required.

Reply:
Thanks, done!